### PR TITLE
Fix Room to Room Transfers to color code and display correct room

### PR DESCRIPTION
**Pull request by OverlordQ**

### CODE REVIEW

File: visuals.js [hunk 1]
```diff
@@ -252,14 +252,18 @@ module.exports = class Visuals {
 			}
 			
 			transactions.forEach(transaction => {
-				const outgoing = transaction.sender.username === room.controller.owner.username;
 				const toSelf = transaction.sender.username === transaction.recipient.username;
-				const colour = outgoing ? '#00FF00' : '#FF0000';
-				const prefix = outgoing ? '+' : '-';
 				let text = '';
+				let colour = '#FFFFFF';
```
Comment (Spedwards) [inline visuals.js]: Perhaps use the resource colour instead if it's a "self trade".

File: visuals.js [hunk 2]
```diff
@@ -252,14 +252,18 @@ module.exports = class Visuals {
...
 				if ( toSelf ) {
-				    text = `${transaction.to} : ${transaction.amount} ${transaction.resourceType}`;
+				    const destRoom = room.name === transaction.to
+				    colour = destRoom ? '#00FF00' : '#FF0000';
```
Comment (Spedwards) [inline visuals.js]: Perhaps use the resource colour instead if it's a "self trade".

File: visuals.js [hunk 3]
```diff
@@ -252,14 +252,18 @@ module.exports = class Visuals {
...
+				    const roomName = destRoom ? transaction.from : transaction.to;
+				    text = `${roomName} : ${transaction.amount} ${transaction.resourceType}`;
```
Comment (Spedwards) [inline visuals.js]: Use `formatNum(transaction.amount)`

**Comment by karlthepagan** [comment]
Please rebase this onto dev.

File: visuals.js (hunk 3 above)
Comment (karlthepagan) [inline visuals.js]: I've started to desire formatNum a lot... dozens of 0's in my float makes me sad

**Comment by OverlordQ** [comment]
Abandoning.

**Pull request closed without merging**